staff/aitor-parra/precourse/bingo/index.js

```diff
@@ -0,0 +1,130 @@
+//BINGO versión 15 números no aleatorios.
+
+// EL CÓDIGO SE EJECUTA CON LA FUNCIÓN ->   Bingo() 
+
+
+var playerCounter = 0;
+var playerS = []
```

Comment:
Suggest not using capital letters at the end of a variable name.

Reply:
Oh yes, I should pay attention to that. **Thank you!**